qa/zcash/full_test_suite.py

```diff
@@ -182,7 +182,7 @@ def rust_test():
     'util-test': util_test,
     'secp256k1': ['make', '-C', repofile('src/secp256k1'), 'check'],
     'univalue': ['make', '-C', repofile('src/univalue'), 'check'],
-    'rpc': [repofile('qa/pull-tester/rpc-tests.py')],
+    'rpc': [repofile('qa/pull-tester/rpc-tests.py'), '--nozmq', '-j32'],
```

Comment:
- The `--nozmq` option should not be set here; it would disable the ZMQ tests in CI, which we don't want.
- The `-j` flag here will cause overcontention on lower-threaded machines (e.g. developers running the script), and will also interact poorly with the RPC tests that internally already parallelize to the full number of threads when making proofs.